diff --git a/node_modules/compute-baseline/dist/baseline/support.js b/node_modules/compute-baseline/dist/baseline/support.js
index 451b8a0..3c1f3b5 100644
--- a/node_modules/compute-baseline/dist/baseline/support.js
+++ b/node_modules/compute-baseline/dist/baseline/support.js
@@ -17,12 +17,7 @@ export function support(feature, browsers) {
             }
         }
         const currentlySupported = unqualifiedReleases.includes(b.current());
-        if (currentlySupported) {
-            support.set(b, lastInitialRelease(unqualifiedReleases));
-        }
-        else {
-            support.set(b, undefined);
-        }
+        support.set(b, lastInitialRelease(unqualifiedReleases));
         logReleaseOmissions(feature, qualifiedReleases, unqualifiedReleases);
     }
     return support;
